files/image_config/caclmgrd/caclmgrd

```diff
@@ -220,6 +222,19 @@ class ControlPlaneAclManager(object):
 
         return block_ip2me_cmds
 
+    def is_rule_ipv4(self, rule_props):
+        if (("SRC_IP" in rule_props and rule_props["SRC_IP"]) or
+            ("DST_IP" in rule_props and rule_props["DST_IP"])):
+            return True
+        else:
+            return False
+
+    def is_rule_ipv6(self, rule_props):
+        if (("SRC_IPV6" in rule_props and rule_props["SRC_IPV6"]) or
+            ("DST_IPV6" in rule_props and rule_props["DST_IPV6"])):
+            return True
+        else:
+            return False
```

Comment:
These functions duplicate the code on lines 365-370. You can replace those lines with calls to these functions.